services/powchain/src/test/java/tech/pegasys/teku/services/powchain/DepositSnapshotsBundleTest.java

```diff
@@ -152,6 +152,7 @@ public static Stream<Arguments> getSupportedNetworks() {
         Arguments.of(Eth2Network.GNOSIS, "https://rpc.ankr.com/gnosis"),
         Arguments.of(Eth2Network.MAINNET, "https://mainnet.infura.io/v3/%INFURA_KEY%"),
         Arguments.of(Eth2Network.SEPOLIA, "https://sepolia.infura.io/v3/%INFURA_KEY%"),
-        Arguments.of(Eth2Network.HOLESKY, "https://holesky.infura.io/v3/%INFURA_KEY%"));
+        Arguments.of(Eth2Network.HOLESKY, "https://holesky.infura.io/v3/%INFURA_KEY%"),
+        Arguments.of(Eth2Network.EPHEMERY, "https://otter.bordel.wtf/erigon"));
```

Comment:
@taxmeifyoucan this is the rpc url I was talking about. Although it's for testing purpose. I just added this https://otter.bordel.wtf/erigon let me know if there is any one I can replace it with

Reply:
I see, this works!